rs/bitcoin/kyt/src/main.rs

```diff
@@ -75,13 +75,23 @@ fn transform(raw: TransformArgs) -> HttpResponse {
 #[ic_cdk::init]
 fn init(arg: KytArg) {
     match arg {
-        KytArg::InitArg(init_arg) => set_config(init_arg.into()),
+        KytArg::InitArg(init_arg) => set_config(Config {
```

Comment:
nit: It might make sense to distinguish between a state initialization and a change of state (meaning have 2 methods, e.g. `initialized_state` and `set_state` that would panic differently) to for example ensure that state is already initialized before doing mutation. Also ok to leave as is.

Reply:
Thanks. I'll leave it as is because I don't think we need to differentiate between the two for the time being.